scripts/hier_mlr.py

```diff
@@ -166,6 +183,8 @@ def __init__(
         self,
         tau: float,
         pool_scale: Optional[float] = None,
+        xi_prior: Optional[float] = None,
```

Comment:
It makes sense to me (mathematically) that `xi_prior` needs to be a float if it's going to make any sense, so I understand this line.